--- node_modules/micro-base/modules/services/http.js	2016-04-13 22:07:35.000000000 +0200
+++ node_modules/seneca-transport/lib/http.js	2016-03-18 21:33:29.000000000 +0100
@@ -122,7 +122,7 @@
           payload: JSON.stringify(data.act)
         }
 
+        Wreck.post(url + '/' + msg.service.replace(/:/g, '/') + '/' + msg.op, requestOptions, function (err, res, payload) {
-        Wreck.post(url, requestOptions, function (err, res, payload) {
           var data = {
             kind: 'res',
             res: payload && _.isObject(payload) ? payload : null,
